cmd.c

```diff
@@ -897,6 +882,75 @@ void cmd_syspage(char *s)
 }
 
 
+void cmd_phfs(char *s)
+{
+	u16 argsc = 0;
+	unsigned int i, major, minor, pos = 0;
+	char args[MAX_CMD_ARGS_NB][LINESZ + 1];
```

Comment:
Since the maximum number of arguments is 4, it can declare 
```
	char args[5][LINESZ + 1]
```
save `5 * 81 ` bytes on the stack.

Reply:
You have right. Fixed